### PR TITLE
docs: add Auth0 SAML Guide

### PR DESCRIPTION
**Pull request by dirien**
Hi,

this PR adds the Guide to use Pulumi SCIM with Auth0

### CODE REVIEW

**Comment by susanev** [comment]
@dirien can we get this added to the nav as well?
nav

if you can drop `Configuring` from the items there while ur there id also appreciate it!

**Comment by dirien** [comment]
@susanev is this `title_tag` or `title`?

**Comment by dirien** [comment]
ok, changed it @susanev. Let se what happens 😅

**Comment by dirien** [comment]
Hi @susanev, I have no idea where this navbar entries coming from... Maybe you have a look.

**Comment by dirien** [comment]
Hi @susanev,

image

Found the issue and fixed it! Now all should be fine 😄

**Comment by lukehoban** [comment]
@dirien @susanev Is this still something we are aiming to land?

cc also @EvanBoyle to make sure this is something we know we are documenting as recommended.

**Comment by dirien** [comment]
@lukehoban, would love to have this added! Thinking to create an AWS Cognito doc too!

File: themes/default/content/docs/guides/saml/auth0.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+title_tag: Configuring Auth0 | SAML SSO
+title: Auth0
+meta_desc: This page provides a walkthrough important aspects of configuring Auth0 as a SAML SSO identity provider (IDP).
```
Comment (ericrudder) [inline themes/default/content/docs/guides/saml/auth0.md:4]:
```suggestion
meta_desc: Configuring Auth0 as a SAML SSO identity provider (IDP).
```

File: themes/default/content/docs/guides/saml/auth0.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    userguides:
+        parent: saml
+        weight: 2
+
+aliases:
+- /docs/reference/service/saml-auth0/
+- /docs/console/accounts/saml/auth0/
+---
+
+This guide walks you through configuring your Auth0 as a SAML SSO identity provider
```
Comment (ericrudder) [inline themes/default/content/docs/guides/saml/auth0.md:15]:
```suggestion
This guide walks you through configuring your Auth0 Authentication Platform as a SAML SSO identity provider
```

File: themes/default/content/docs/guides/saml/auth0.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+(IDP) for the Pulumi Service.
+
+## Prerequisites
+
+* Your organization must already be configured to use [SAML](/docs/guides/saml/sso/) with Pulumi.
+* You must be an admin of your Pulumi organization.
+* (Optional, but highly recommended) You should have more than one admin for your Pulumi organization.
+
+## Enabling SAML For Your Auth0 Application
```
Comment (ericrudder) [inline themes/default/content/docs/guides/saml/auth0.md:24]:
```suggestion
## Enabling SAML For Your Auth0 Authentication Platform
```

File: themes/default/content/docs/guides/saml/auth0.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+To enable SAML for your Auth0 application, navigate to the **Applications** section of your Auth0 dashboard. You may
```
Comment (ericrudder) [inline themes/default/content/docs/guides/saml/auth0.md:26]:
```suggestion
To enable SAML for your Auth0 Authentication Platform, navigate to the **Applications** section of your Auth0 dashboard. You may
```

File: themes/default/content/docs/guides/saml/auth0.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+need to create a new application or select an existing application. Next we need to get the SAML metadata XML to
```
Comment (ericrudder) [inline themes/default/content/docs/guides/saml/auth0.md:27]:
```suggestion
need to create a new application or select an existing application. Next, we need to get the SAML metadata XML to
```

File: themes/default/content/docs/guides/saml/auth0.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+the browser or a command line tool like `curl`.
+
+![Auth0 Application Endpoints](/images/docs/reference/service/saml-auth0/auth0-app-endpoints.png)
+
+Paste the contents of the downloaded XML file in the **Change Membership Requirements** dialog for setting up Pulumi
+SAML SSO.
+
+![Pulumi SAML SSO Configuration](/images/docs/reference/service/saml-auth0/auth0-saml-sso-config.png)
+
+Lastly, navigate in the **Settings** tab of your application to the **Application URIs** section. In the **Application
```
Comment (ericrudder) [inline themes/default/content/docs/guides/saml/auth0.md:44]:
```suggestion
Finally, navigate in the **Settings** tab of your application to the **Application URIs** section. In the **Application
```

**Comment by ericrudder** [comment]
I left a few grammatical comments, however, this looks like we should
land!  Thanks @Engin Diri ***@***.***>

**Comment by dirien** [comment]
Thanks @ericrudder for your feedback! Appreciate this very much! 👍 

So this PR is ready, merged all suggestion!

**Review by djgrove — APPROVED** [review]

**Pull request merged**